docs/fast_data/configuration/projection_storer.md

```diff
@@ -0,0 +1,858 @@
+---
+id: projection_storer
+title: Projection Storer Configuration
+sidebar_label: Projection Storer
+---
+
+Projection Storer is the service in charge of constantly keeping up-to-date projections records with respect to change events
+that are received from the associated System of Records. Once modifications are stored to the database, the service
+triggers downstream components with the proper mechanism, so that Single Views can be regenerated with latest data. 
+
+This service partially overlap with the concerns of Real-Time Updater plugin, in particular converting in near real-time
```

Comment:
questo pezzo qua è da mettere in highlight, non so se in un :::note o qualcos'altro. Però è la prima cosa che l'utente deve leggere. Insieme alla guida di migrazione aggiungerei poi una FAQ che risponde alla domanda "Why using Projection Storer instead of Real-Time Updater

Reply:
Done